tools/copyblocks/main.go

```diff
@@ -157,54 +169,54 @@ func main() {
 }
 
 func initializeBuckets(ctx context.Context, cfg config) (sourceBucket bucket, destBucket bucket, err error) {
+	if cfg.sourceService == "" || cfg.destinationService == "" {
+		return nil, nil, errors.New("--source-service or --destination-service is missing")
+	}
```

Comment:
I know this is what is done already below for the source bucket and destination bucket options, but should we split this check in two so we can be clear which option is missing?

Reply:
Yeah this has bothered me too in the past, swapped it to check each individually.